docs/_src/api/api/document_store.md

```diff
@@ -985,7 +985,7 @@ the vector embeddings are indexed in a FAISS Index.
 #### \_\_init\_\_
 
 ```python
- | __init__(sql_url: str = "sqlite:///", vector_dim: int = 768, faiss_index_factory_str: str = "Flat", faiss_index: Optional["faiss.swigfaiss.Index"] = None, return_embedding: bool = False, index: str = "document", similarity: str = "dot_product", embedding_field: str = "embedding", progress_bar: bool = True, duplicate_documents: str = 'overwrite', **kwargs, ,)
+ | __init__(sql_url: str = "sqlite:///faiss_document_store.db", vector_dim: int = 768, faiss_index_factory_str: str = "Flat", faiss_index: Optional["faiss.swigfaiss.Index"] = None, return_embedding: bool = False, index: str = "document", similarity: str = "dot_product", embedding_field: str = "embedding", progress_bar: bool = True, duplicate_documents: str = 'overwrite', **kwargs, ,)
```

Comment:
why is doc and api change now in this PR? Lets move it to a separate one?

Reply:
These are changes by our bot. I will try to remove them. they were added automatically and are by now already merged on master branch by Branden.

Reply:
I talked to Branden (who talked to Markus): suggestion is to keep these changes in the branch as they are meaningful updates to our docs. They are investigating further. Happened also with other PRs, e.g., https://github.com/deepset-ai/haystack/pull/1485/files